umm-spec-lib/src/cmr/umm_spec/test/dif10_expected_conversion.clj

```diff
@@ -336,10 +337,25 @@
   (when-let [media (first media)]
     [media]))
 
+(defn- expected-dist-parsed-format
+  "I could not figure out how to not create this function using the update call in the expand-file-dist-infos function below. So this function exists to replace the new value with the old."
+  [old-value new-value]
+  new-value)
```

Comment:
I'm confused by this function. It looks like it isn't even being given `new-value` when it's called below. What is going on here?

Reply:
I was able to use assoc instead of update in the function below and therefore I don't need this function anymore.  I have deleted it.